src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -38,6 +38,11 @@ public sealed class UIActionEngine
         private readonly NuGetPackageManager _packageManager;
         private readonly INuGetLockService _lockService;
 
+        /// <summary>
+        /// Property is referenced by the Preview Window.
+        /// </summary>
+        public bool IsPackageSourceMappingEnabled { get; private set; }
```

Comment:
It is used in this branch, although it's main purpose will be to serve as a XAML binding in the next PR (referred to in this PR's description).
Since I moved where the PackageSourceMappings are loading from settings, and since the telemetry event needs to use this flag, I wanted to start using this property now. I agree it is less necessary to use a public property in this PR so technically we could have a middle-ground commit that does the refactoring above without a new property, and then my next PR moves it to a new property. I like the idea of doing the change now since it's tangentially relevant.